### PR TITLE
readme updated

### PR DESCRIPTION
**Pull request by guillaum3f**
There is a new folder at root : hack.ether

### CODE REVIEW

**Comment by dannywillems** [comment]
Could you explain what is the purpose of this PR?

File: README.md [hunk 1]
```diff
@@ -1,10 +1,15 @@
 Tech blog of hackerhouse.paris
 ==============================
 
-This blog shows the posts of hackerhouse.paris .
+## Hackaton Ethereum
+
+Git Clone(http://github.io)
```
Comment (dannywillems) [inline README.md]: ? What do you want to do?

File: README.md [hunk 2]
```diff
@@ -1,10 +1,15 @@
 Tech blog of hackerhouse.paris
 ==============================
 
-This blog shows the posts of hackerhouse.paris .
```
Comment (dannywillems) [inline README.md]: Why did you remove this line?

**Comment by n1c01a5** [comment]
I don't understand this PR. @guillaum3f can you explain me?
